Lib/email/utils.py

```diff
@@ -345,6 +346,8 @@ def localtime(dt=None, isdst=-1):
     to divine whether summer time is in effect for the specified time.
 
     """
+    if isdst != -1:
+        warnings.warn("The isdst parameter to localtime is deprecated.", DeprecationWarning, stacklevel=2)
```

Comment:
There's a new API we can use for this, see https://discuss.python.org/t/introducing-warnings-deprecated/14856

If this is merged in time for [Python 3.11 beta (2022-05-06)](https://peps.python.org/pep-0664/#schedule), then it can be removed in 3.13.

(Six years ago, https://github.com/python/cpython/issues/72346#issuecomment-1093726798 suggested a single release from deprecation to removal, but I think unless a core developer requests it, we're probably safer sticking to the standard two releases.)

So for example:

```suggestion
        warnings._deprecad("The isdst parameter to localtime", remove=(3, 13))
```